### PR TITLE
Add static routes for blog posts

### PR DESCRIPTION
**Pull request by denisw**
I took the opportunity to clean up the `next.config.js` file a bit. Hope it's an improvement.

### CODE REVIEW

**Comment by torkjels** [comment]
@denisw the netlify build failed.
image

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 0bb542a132d671b79a2b27620b1bc1f778360c46

https://deploy-preview-150--masifunde.netlify.com

**Comment by torkjels** [comment]
Any particular reason you're not adding the locale to the path of the blog posts?
So, doing ```pathMap[`/${locale}/blog/${slug}`] =``` instead of ```pathMap[`/blog/${slug}`] =```.

File: next.config.js [hunk 1]
```diff
@@ -1,43 +1,115 @@
...
+    host: env.CONTENTFUL_HOST,
+  })
+
+  let posts = []
+  let skip = 0
+  let fetchResult
+
+  // The Contentful API only allows getting 1000 items at a time.
+  // For the case where there are more posts, we need to do multiple
+  // requests.
```
Comment (torkjels) [inline next.config.js]: Nice explanation, very much needed I think 👍

**Pull request merged**